internal/topo/planner/logicalPlan.go

```diff
@@ -27,10 +33,44 @@ type LogicalPlan interface {
 	PruneColumns(fields []ast.Expr) error
 }
 
+type ExplainInfo interface {
+	Type() string
+	Explain() string
+	BuildExplainInfo(id int64)
+}
+
+type PlanExplainInfo struct {
+	T    PlanType `json:"type"`
+	Info string   `json:"info"`
+	id   int64    `json:"id"`
```

Comment:
```suggestion
	ID   int64    `json:"id"`
```

Reply:
I want use indentation to represent hierarchical relationships.Because I want to marshal column of ID,  so I use 
"ID   int64    `json:"-"`"